guides/micronaut-openapi-generator-server/java/src/main/java/example/micronaut/library/Application.java

```diff
@@ -0,0 +1,9 @@
+package example.micronaut.library;
+
+import io.micronaut.runtime.Micronaut;
+
+public class Application {
```

Comment:
delete this class. during the build, an application is created and your source code is copied into that project. if you need to replace an auto-generated class like this because you've changed something, include it in the guide so it overwrites the generated file, otherwise there's no need